### PR TITLE
README: Rework to start using GitHub for development

### PR DESCRIPTION
**Pull request by otavio**
This changes the content so we start sending people to GitHub and
deprecate the OpenEmbedded-Devel mailing list as the place for patch
submission.

The content has been reworked so it renders better inside GitHub and
provide a more structured content.

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+====================================
+
+This layer depends on:
+
+URI: git://github.com/openembedded/oe-core.git
+branch: master
+revision: HEAD
+
+When building stuff like qtdeclarative, qtquick, qtwebkit, make sure that
+you have required PACKAGECONFIG options enabled in qtbase build, see qtbase.inc
```
Comment (shr-project) [inline README.md]: qtbase.inc doesn't exist anymore, should probably say just qtbase recipe.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+for detail.
+
+
+Contributing
+------------
+
+Please submit any patches against the meta-qt5 layer by using the GitHub
+pull-request feature.  Fork the repo, make a branch, do the work, pull from
```
Comment (shr-project) [inline README.md]: Should we say "rebase on latest target branch in upstream" instead of just pull from upstream?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+Qt5 OpenEmbedded/Yocto Project layer
+====================================
+
+This layer depends on:
+
+URI: git://github.com/openembedded/oe-core.git
```
Comment (shr-project) [inline README.md]: openembedded/openembedded-core.git

File: README.md (hunk 2 above)
Comment (otavio) [reply README.md]: Ok, I made a small change and will update.

**Pull request merged**